app/questionnaire/questionnaire_schema.py

```diff
@@ -1249,6 +1269,9 @@ def get_item_anchor(self, section_id: str, list_name: str) -> str | None:
                 if item["for_list"] == list_name and item.get("item_anchor_answer_id"):
                     return f"#{str(item['item_anchor_answer_id'])}"
 
+    def is_block_in_repeating_blocks(self, block_id: str) -> bool:
```

Comment:
Is this ever being used outside tests?

Reply:
Great point, I made this in an earlier iteration and guess we don't need it now!

Reply:
Done 👍